### DISCUSSION TITLE
Issue w/ Pressure Diffusion Example Output

### DISCUSSION THREAD

**Discussion by ghost** [idaholab/moose#17983] [Q&A Getting Started] [answered]
I'm currently trying to work through the application development tutorial from the MOOSE framework website on my device with Ubuntu OS. Specifically, I'm trying to run the pressure diffusion example. I have used both the code displayed on the website and the code provided in the MOOSE workshop slides, but neither of them display a result resembling the one displayed on the MOOSE website:
![step02_result](https://user-images.githubusercontent.com/79124140/120383247-46bf2580-c2ea-11eb-9455-47834f3112e6.png)
Instead, Peacock is displaying this after execution:
![Screenshot from 2021-06-01 15-03-34](https://user-images.githubusercontent.com/79124140/120383508-930a6580-c2ea-11eb-9c9c-5baf247707ff.png)

Has anyone else experienced a similar issue, or know what I can try to fix it? I can provide the input file if necessary.
Thanks!

Edit: This error does not exist when I perform the simulation using the darcy_thermo_mech directory within moose/tutorials that comes with installation. This error was a result from attempting to perform the same simulation using my own developed application, so I assume there is an issue in the way I setup my application originally.

**Comment by GiudGiud** [comment, marked as answer]
Hello

Are you talking about the darcy_thermo_mech example?
Which step of the workshop slides?

Best,
Guillaume

**Reply by ghost** [reply]
Yes, I'm speaking of that one; which is on Step 1: Geometry and Diffusion in the workshop slides (slide 7). Sorry for not specifying earlier!

**Reply by GiudGiud** [reply]
no worries.
I tried it just now in tutorials/darcy_thermo_mech/step01_diffusion/problems and it looks just like in the top picture in peacock.
Can you paste your input? I ll look if it differs from the code in the repository

**Comment by crswong888** [comment]
@speyres, you probably just need to rescale the contours to render it like the image in the tutorial. Double check the contour limits you used when rendering this image (I think there are settings "min" and "max" in Peacock, set these to the values for the BCs at the right (0 Pa) and left (4000 Pa) of the pipe, respectively).
